### DISCUSSION TITLE
Cannot load repository

### DISCUSSION THREAD

**Discussion by AfeworkM** [samapriya/awesome-gee-community-datasets#68] [General]
Dear Samapriya, Your contribution on this platform is very big especially for those implementing  sustainable land management  in 3rd world in east Africa, still enjoying the resources that you made available but sample code couldn't load in my repository on GEE, what is the reason to have sample code on my code list.

with best regards

**Comment by samapriya** [comment]
Hi @AfeworkM 
Can you try the links again I have made it available there are some major updates coming so I was testing repo structures. It should be accessible now . I just confirmed it with a colleague

**Comment by AfeworkM** [comment]
I checked it so many times for different themes, it says like that.

with big regards

On Fri, 30 Sept 2022 at 20:17, Samapriya Roy ***@***.***>
wrote:

> Hi @AfeworkM <https://github.com/AfeworkM>
> Can you try the links again I have made it available there are some major
> updates coming so I was testing repo structures. It should be accessible
> now . I just confirmed it with a colleague
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/samapriya/awesome-gee-community-datasets/discussions/68#discussioncomment-3774142>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASZC27AOUEJFV4G5UNRCCKDWA4OD3ANCNFSM6AAAAAAQZ6A2VA>
> .
> You are receiving this because you were mentioned.Message ID:
> <samapriya/awesome-gee-community-datasets/repo-discussions/68/comments/3774142
> @github.com>
>


-- 

Afework Mekeberiaw

Specialist / GIS and Remote Sensing

Sustainable Land Management Programme
Ministry of Agriculture, Addis Ababa,
Ethiopia

Mob: +251-9 <%2B%2B251-923023576>11675804
Skype: afotesfaye

**Comment by AfeworkM** [comment]
it says
[image: image.png]

On Fri, 30 Sept 2022 at 20:58, Afework Mekeberiaw Worku <
***@***.***> wrote:

> I checked it so many times for different themes, it says like that.
>
> with big regards
>
> On Fri, 30 Sept 2022 at 20:17, Samapriya Roy ***@***.***>
> wrote:
>
>> Hi @AfeworkM <https://github.com/AfeworkM>
>> Can you try the links again I have made it available there are some major
>> updates coming so I was testing repo structures. It should be accessible
>> now . I just confirmed it with a colleague
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/samapriya/awesome-gee-community-datasets/discussions/68#discussioncomment-3774142>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ASZC27AOUEJFV4G5UNRCCKDWA4OD3ANCNFSM6AAAAAAQZ6A2VA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> <samapriya/awesome-gee-community-datasets/repo-discussions/68/comments/3774142
>> @github.com>
>>
>
>
> --
>
> Afework Mekeberiaw
>
> Specialist / GIS and Remote Sensing
>
> Sustainable Land Management Programme
> Ministry of Agriculture, Addis Ababa,
> Ethiopia
>
> Mob: +251-9 <%2B%2B251-923023576>11675804
> Skype: afotesfaye
>


-- 

Afework Mekeberiaw

Specialist / GIS and Remote Sensing

Sustainable Land Management Programme
Ministry of Agriculture, Addis Ababa,
Ethiopia

Mob: +251-9 <%2B%2B251-923023576>11675804
Skype: afotesfaye

**Comment by AfeworkM** [comment]
Dear Samapriya/Awesome-Gee-Community-Datasets

Big thank you , now it is coming.please keep supporting the geospatial
community and contributing big to scientific community.

God bless you and your country.
best regards

On Fri, 30 Sept 2022 at 20:59, Afework Mekeberiaw Worku <
***@***.***> wrote:

> it says
> [image: image.png]
>
> On Fri, 30 Sept 2022 at 20:58, Afework Mekeberiaw Worku <
> ***@***.***> wrote:
>
>> I checked it so many times for different themes, it says like that.
>>
>> with big regards
>>
>> On Fri, 30 Sept 2022 at 20:17, Samapriya Roy ***@***.***>
>> wrote:
>>
>>> Hi @AfeworkM <https://github.com/AfeworkM>
>>> Can you try the links again I have made it available there are some
>>> major updates coming so I was testing repo structures. It should be
>>> accessible now . I just confirmed it with a colleague
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/samapriya/awesome-gee-community-datasets/discussions/68#discussioncomment-3774142>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/ASZC27AOUEJFV4G5UNRCCKDWA4OD3ANCNFSM6AAAAAAQZ6A2VA>
>>> .
>>> You are receiving this because you were mentioned.Message ID:
>>> <samapriya/awesome-gee-community-datasets/repo-discussions/68/comments/3774142
>>> @github.com>
>>>
>>
>>
>> --
>>
>> Afework Mekeberiaw
>>
>> Specialist / GIS and Remote Sensing
>>
>> Sustainable Land Management Programme
>> Ministry of Agriculture, Addis Ababa,
>> Ethiopia
>>
>> Mob: +251-9 <%2B%2B251-923023576>11675804
>> Skype: afotesfaye
>>
>
>
> --
>
> Afework Mekeberiaw
>
> Specialist / GIS and Remote Sensing
>
> Sustainable Land Management Programme
> Ministry of Agriculture, Addis Ababa,
> Ethiopia
>
> Mob: +251-9 <%2B%2B251-923023576>11675804
> Skype: afotesfaye
>


-- 

Afework Mekeberiaw

Specialist / GIS and Remote Sensing

Sustainable Land Management Programme
Ministry of Agriculture, Addis Ababa,
Ethiopia

Mob: +251-9 <%2B%2B251-923023576>11675804
Skype: afotesfaye
